lib/kintsugi/cli.rb

```diff
@@ -199,6 +202,10 @@ def create_root_command
           Settings.allow_duplicates = true
         end
 
+        unless options[:"interactive-resolution"].nil?
+          Settings.interactive_resolution = options[:"interactive-resolution"]
+        end
```

Comment:
Shouldn't this be according to whether a tty is available?

Reply:
This is handled in `Kintsugi::Settings`.